libs/ledger/include/ledger/consensus/consensus.hpp

```diff
@@ -63,17 +65,19 @@ class Consensus final : public ConsensusInterface
 
 private:
   static constexpr std::size_t HISTORY_LENGTH = 1000;
+  static constexpr uint64_t    INVALID_BLOCK  = 0xFFFFFFFFFFFFFFFFllu;
```

Comment:
minor: could use std::numeric_limits ... max

Reply:
done not needed as part of this PR